client/src/components/modal-forms/PhaseDialog.js

```diff
@@ -15,18 +18,44 @@ const useStyles = makeStyles(() => ({
   formRow: {
     gap: '25px',
   },
+  list: {
+    overflow: 'scroll',
+    maxHeight: '250px',
+  },
+  listItem: {
+    fontSize: '14px',
+    paddingTop: '0',
+    paddingBottom: '0',
+  },
 }));
 
-export const PhaseDialog = ({ onSubmit, onClose, open, content, isNew }) => {
+export const PhaseDialog = ({ onSubmit, onClose, open, content, isNew = false }) => {
```

Comment:
I would've thought `isNew` would be defaulted to true, but I don't think it matters too much

Reply:
`PhaseDialog` is being referenced in two spots, currently, it is easier to add additional props/flags to the new reference as its being rendered in the JSX whereas the edit reference is being passed as an object to the new DataTable component, so would require some updates to DataTable. For that reason, I've defaulted the flag to false (It was reading as undefined prior)